VMs/metrics/ubuntu16.04/user-data

```diff
@@ -0,0 +1,54 @@
+#cloud-config
+# Copyright (c) 2018 Intel Corporation
+# 
+# SPDX-License-Identifier: Apache-2.0
+
+groups:
+  - jenkins
+
+users:
+  - name: USERNAME
+    gecos: Jenkins User
+    lock-passwd: false
+    shell: /bin/bash
+    sudo: ALL=(ALL) NOPASSWD:ALL
+    ssh-authorized-keys:
+      - PUBLICKEYCONTENTS
+
+packages:
+ - gawk
+ - bc
+ - curl
+ - git
+ - jq
+ - default-jre
+ - gcc
+ - make
+ - build-essential
+
+runcmd:
+
+ - rm /etc/update-motd.d/10-help-text /etc/update-motd.d/51-cloudguest
+ - rm /etc/update-motd.d/90-updates-available
+ - rm /etc/legal
```

Comment:
OOI why?

Reply:
I inherited it from https://github.com/intel/ccloudvm/blob/master/workloads/clearcontainers-3.x.yaml#L46
I suspect it has to do with the rules for modifying and 'shipping' an Ubuntu image.
@markdryan @mcastelino - are you able to clarify for us?

Reply:
This came from the original ccloudvm workload, ciao.  This workload displays a prompt every time you log into the VM explaining how to start running the ciao development environment.  These three files where removed, as their presence caused the ciao start up instructions to get lost in a whole pile of other information.

Reply:
Thanks @markdryan - in which case I think i can drop those lines.